backend/src/main/scala/cromwell/backend/RuntimeEnvironment.scala

```diff
@@ -23,20 +26,20 @@ object RuntimeEnvironmentBuilder {
         callRoot.resolve(s"tmp.$hash").pathAsString
       }
 
-      val cores: Int = CpuValidation.instanceMin.validate(runtimeAttributes).getOrElse(minimums.cores)
+      val cores: Int Refined Positive = CpuValidation.instanceMin.validate(runtimeAttributes).getOrElse(minimums.cores)
 
       val memoryInMiB: Double =
         MemoryValidation.instance().
           validate(runtimeAttributes).
-          map(_.to(MemoryUnit.MiB).amount).
+          map(_.toMebibytes).
```

Comment:
Not sure if this was converted to `MiB` (and not `MB`) on purpose, left it as is in case it was